fpga_builder/utils.tcl

```diff
@@ -87,6 +87,16 @@ proc build {proj_name top_name proj_dir} {
 
   set output_dir [file normalize $proj_dir/../output]
 
+  puts "usr_access value: $usr_access"
+  set design_version [string range "$usr_access" 2 7]
```

Comment:
Tcl shouldn't be aware of the formatting of the bits.  If we want to print it out to verify, can do that in the python before calling the tcl.  With this setup, refactoring later would be a pain in the butt.